Content.Shared/Doors/Systems/SharedFirelockSystem.cs

```diff
@@ -102,43 +77,9 @@ private void OnAfterPried(EntityUid uid, FirelockComponent component, ref PriedE
 
     #endregion
 
-    #region Visuals
-
-    private void UpdateVisuals(EntityUid uid, FirelockComponent component, EntityEventArgs args) => UpdateVisuals(uid, component);
-
-    private void UpdateVisuals(EntityUid uid,
-        FirelockComponent? firelock = null,
-        DoorComponent? door = null,
-        AppearanceComponent? appearance = null)
-    {
-        if (!Resolve(uid, ref door, ref appearance, false))
-            return;
-
-        // only bother to check pressure on doors that are some variation of closed.
-        if (door.State != DoorState.Closed
-            && door.State != DoorState.Welded
-            && door.State != DoorState.Denying)
-        {
-            _appearance.SetData(uid, DoorVisuals.ClosedLights, false, appearance);
-            return;
-        }
-
-        if (!Resolve(uid, ref firelock, ref appearance, false))
-            return;
-
-        _appearance.SetData(uid, DoorVisuals.ClosedLights, firelock.IsLocked, appearance);
-    }
-
-    #endregion
-
```

Comment:
Check for regression on orange lights when locked/active.

Reply:
What regression?

Reply:
Opening/closing the door always sets lights to green. If the door is open or closed the lights should be off or when IsLocked they should be orange.